velox/functions/prestosql/ArrayContains.cpp

```diff
@@ -217,13 +246,21 @@ class ArrayContainsFunction : public exec::VectorFunction {
                 .argumentType("T")
                 .build()};
   }
+
+ private:
+  bool throwOnNestedNull_;
 };
 
 } // namespace
 
 VELOX_DECLARE_VECTOR_FUNCTION(
     udf_array_contains,
     ArrayContainsFunction::signatures(),
-    std::make_unique<ArrayContainsFunction>());
+    std::make_unique<ArrayContainsFunction>(true));
+
+VELOX_DECLARE_VECTOR_FUNCTION(
+    udf_$internal$contains,
```

Comment:
Can you add a test case for this ? Thanks !

Reply:
Also do we have documentation regarding these internal functions ?

Reply:
We don't have documentation about the internal functions because they are "internal" that we don't want users to rely on. I added a comment explaining what this $internal$contains do.